test/specs/controller.scatter.test.js

```diff
@@ -3,6 +3,43 @@ describe('Chart.controllers.scatter', function() {
 		expect(typeof Chart.controllers.scatter).toBe('function');
 	});
 
+	it('should test default tooltip callbacks', function() {
+		var chart = window.acquireChart({
+			type: 'scatter',
+			data: {
+				datasets: [{
+					data: [{
+						x: 10,
+						y: 15
+					}],
+					label: 'dataset1'
+				}],
+			},
+			options: {}
+		});
+		var meta = chart.getDatasetMeta(0);
+		var point = meta.data[0];
+		var node = chart.canvas;
+		var rect = node.getBoundingClientRect();
+		var evt = new MouseEvent('mousemove', {
```

Comment:
Can you use `jasmine.triggerMouseEvent(chart, 'mousemove', point);` instead?

Reply:
Absolutely! Thanks for the tip. Latest commit uses it.